src/components/LHNOptionsList/OptionRowLHNData.tsx

```diff
@@ -36,6 +38,11 @@ function OptionRowLHNData({
     const optionItemRef = useRef<OptionData>();
 
     const shouldDisplayViolations = canUseViolations && ReportUtils.shouldDisplayTransactionThreadViolations(fullReport, transactionViolations, parentReportAction);
+    const shouldDisplayReportViolations =
```

Comment:
I think we'll need to modify who sees this.

Essentially, we just want the RBR to be shown for the submitter of this specific expense report.

However, there's a couple edge cases here
- Anyone can be added to the expense chat report, in those cases we don't want the RBR to be shown because they are not the submitter
- An admin could also have an expense chat report where they are the submitter to a different approver

You might be able to use the ownerReportID on the report onyx data